omnidocs/tasks/math_expression_extraction/extractors/surya_math.py

```
@@ -0,0 +1,189 @@
+from pathlib import Path
+from typing import Union, List, Dict, Any, Optional
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)